src/index.ts

```diff
@@ -149,7 +147,7 @@ async function main (): Promise<void> {
       yamux(),
       mplex()
```

Comment:
Maybe remove mplex?

Reply:
fixed in [`4e48bac` (#139)](https://github.com/libp2p/js-libp2p-amino-dht-bootstrapper/pull/139/commits/4e48bac1779267f17f77c5d13a439f17cf89b297)